docs/source/dummies_the_database.rst

```diff
@@ -8,17 +8,17 @@ Let's do this now. First, open up the terminal and navigate to the directory con
 
 Table/Object Duality
 --------------------
 
 In quantum mechanics there is a phenomenon known as wave/particle duality. It basically means that various sub-atomic particles, depending on the conditions, can behave like either a particle or a wave. This was a big deal to physicists because for centuries it had been assumed that things could either be a particle or a wave, but not both. By the late 19th century people were pretty confident that things like electrons were particles, though there had been vigorous debate over whether light was a particle or a wave. In the end it turns out the answer is "both, sort of": depending how you observe them, light "photons" will behave like particles or like a wave, and the conclusion, therefore, is that they're actually something else that has properties of both particles and waves.
 
-The point of this digression is to get you to think about how a single thing can have a dual nature. From glancing at the tables you can see that certain objects (like a `node`, for instance) can be stored as a row in a table. Because of this, anything you can do with a row of data you can do with nodes. For instance you can look them up by a particular value (not the most exciting stuff I admit). But when your experiment is running, this row of data in a table in the database will have properties that seem to go beyond what a simply row of data can do. In some contexts, entities like nodes
+The point of this digression is to get you to think about how a single thing can have a dual nature. From glancing at the tables you can see that certain objects (like a `node`, for instance) can be stored as a row in a table. Because of this, anything you can do with a row of data you can do with nodes. For instance you can look them up by a particular value (not the most exciting stuff I admit). But when your experiment is running, this row of data in a table in the database will have properties that seem to go beyond what a simple row of data can do. In some contexts, entities like nodes
 will also include `behavior`.
 
-Let's illustrate this with an example. Imagine you want to know the creation time of the network that contains the node with id 5. This might sound a little daunting, but it you rootle around in Postico you'll see that the node table contains a column called `network_id` and the network table contains a column called `creation_time`. If we approach this problem thinking in terms of tables, here's a rough description of how we could solve it:
```

Comment:
I also had not encountered the word "rootle" before, but confirmed it exists outside of Tom's personal vocabulary. :)

Reply:
yes, I thought that perhaps a synonym would work better there :)

Reply:
ha, guess it must be a Britishism? "Dig" will probably suffice as a replacement.

Reply:
hehe, most likely. Depends on the target audience I guess, but as a native (non-British) speaker I have not heard the word before